### ISSUE TITLE
Make GitHub Actions work for complete build

### ISSUE THREAD

**Issue by jskeet** [googleapis/google-cloud-dotnet#5493]
Currently we're running out of disk space. The size of Grpc.Core may well be relevant here, which is being improved. We can probably tidy up as we go to reduce this though.

**Comment by jskeet** [comment]
Directory size after local build: 50.6GB.

After removing these files:

- libgrpc_csharp_ext.x64.dylib
- libgrpc_csharp_ext.x86.dylib
- grpc_csharp_ext.x64.dll
- grpc_csharp_ext.x86.dll
- libgrpc_csharp_ext.x86.so

... it's down to 24.9GB, but that's still too big. Hmm.

**Comment by jskeet** [comment]
Now trying "let's remove the bin directory for desktop framework builds entirely after building each API". This may well be enough, given that the desktop framework builds are the huge ones.

**Comment by jskeet** [comment]
From a log

```text
Disk space at start of run:
Filesystem     1K-blocks     Used Available Use% Mounted on
udev             3542252        0   3542252   0% /dev
tmpfs             712132      680    711452   1% /run
/dev/sda1       87218124 66817908  20383832  77% /
tmpfs            3560644        8   3560636   1% /dev/shm
tmpfs               5120        0      5120   0% /run/lock
tmpfs            3560644        0   3560644   0% /sys/fs/cgroup
/dev/sda15        106858     3685    103174   4% /boot/efi
/dev/sdb1       14382088  4235296   9396508  32% /mnt
```

**Comment by jskeet** [comment]
This was fixed by #5498.
